sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -526,43 +585,50 @@ public void setWrapperSdkListener(WrapperSdkListener wrapperSdkListener) {
     }
 
     @VisibleForTesting
-    private synchronized void handleUserConfirmation(@UserConfirmationDef int userConfirmation) {
+    private synchronized void handleUserConfirmation(@UserConfirmationDef final int userConfirmation) {
         if (mChannel == null) {
             MobileCenterLog.error(LOG_TAG, "Crashes service not initialized, discarding calls.");
             return;
         }
 
-        if (userConfirmation == DONT_SEND) {
+        mHandler.post(new Runnable() {
```

Comment:
Not a big deal but it is just using same handler if it is already on background thread.